### ISSUE TITLE
Feature Request: adding class_weights to tfLite Model Maker

### ISSUE THREAD

**Issue by pdubz-sudo** [tensorflow/tensorflow#48703]
## Description of issue (what needs changing):
It would be great if somebody could add class_weights to the `model maker` .create() arguments because it's pretty common to work with imbalanced datasets in real life. I'm trying to retrain a model with model maker for image classification and have about 15 labels with very different balances in classes. I'm trying to change the code myself and hoping to do a pull request but I can't my code to work.

### Clear description

For example, why should someone use this method? How is it useful?
Many datasets are imbalanced and having class_weights in the .create() method will help a lot to tackle that issue.

### Correct links

https://github.com/tensorflow/examples/tree/master/tensorflow_examples/lite/model_maker/core/task

### Parameters defined

Description which could be added.
class_weight: : Optional dictionary mapping class indices (integers) to a 
        weight (float) value, used for weighting the loss function 
        (during training only). This can be useful to tell the model to 
        "pay more attention" to samples from an under-represented class.

Are you planning to also submit a pull request to fix the issue? See the docs

I'm trying to fix it myself and would like to do a pull request but my build never works after I changed the code. I've been trying to fix it for a while and finally just figured I should see if somebody else knows how to do it.

**Comment by abattery** [comment]
@lintian06 could you review this feature request?

**Comment by wee-analyze** [comment]
I made some code which I think adds the feature. I'm not sure if it works though because I can't build from source when I follow the instructions from the model_maker readme. I make a fork and added the changes but I don't know how to make a pull request to TensorFlow. I tried, but it doesn't seem to show up on the TensorFlow PRs.

This is the fork where I made the changes [https://github.com/wee-analyze/examples/pull/1](url)

**Comment by wee-analyze** [comment]
I figured out how to make a PR in the TensorFlow repo https://github.com/tensorflow/tensorflow/issues/48703#issuecomment-825221404

**Comment by wee-analyze** [comment]
I changed some code which I think adds the feature. I'm not sure if it works though because I can't build from source when I follow the instructions from the model_maker readme (not sure why that is).  This is the PR I made [https://github.com/tensorflow/examples/pull/307](https://github.com/tensorflow/examples/pull/307)

**Comment by wee-analyze** [comment]
@lintian06 I changed some code which I think adds the feature. If you're interested, maybe you could check my code? I'm not sure if it works because I can't build from source when I follow the instructions from the model_maker readme (not sure why that is).  This is the PR I made [https://github.com/tensorflow/examples/pull/307](https://github.com/tensorflow/examples/pull/307)

**Comment by tilakrayal** [comment]
Hi,

Thank you for opening this issue. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.

The Tensorflow team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TensorFlow [version](https://pypi.org/project/tensorflow/) with the latest compatible hardware configuration which could potentially resolve the issue. If you are still facing the issue, please create a new GitHub issue with your latest findings, with all the debugging information which could help us investigate.

Please follow the [release notes](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md) to stay up to date with the latest developments which are happening in the Tensorflow space.
